src-ts/tools/learn/course-details/CourseDetailsPage.tsx

```diff
@@ -25,13 +27,14 @@ interface CourseDetailsPageProps {
 
 const CourseDetailsPage: FC<CourseDetailsPageProps> = (props: CourseDetailsPageProps) => {
     const routeParams: Params<string> = useParams()
+    const { profile }: ProfileContextData = useContext(profileContext)
 
     const {
         course,
         ready,
     }: CoursesProviderData = useCoursesProvider(routeParams.certification)
 
-    const { progress }: MyCertificationProgressProviderData = useMyCertificationProgress(routeParams.certification)
+    const { certificateProgress: progress }: MyCertificationProgressProviderData = useMyCertificationProgress(profile?.userId, routeParams.certification)
```

Comment:
I had to change the URL scheme to use unique IDs instead of the user ID + certification name so that each certification progress record is unique. We should probably just make that update in this PR now as that's what the API on dev expects.